api/src/services/email.service.ts

```diff
@@ -23,33 +23,34 @@ import {
 } from '@prisma/client';
 import { IdDTO } from '../dtos/shared/id.dto';
 import { Listing } from '../dtos/listings/listing.dto';
-import { SendGridService } from './sendgrid.service';
 import { ApplicationCreate } from '../dtos/applications/application-create.dto';
 import { User } from '../dtos/users/user.dto';
 import Unit from '../dtos/units/unit.dto';
 import { getPublicEmailURL } from '../utilities/get-public-email-url';
+
 dayjs.extend(utc);
 dayjs.extend(tz);
 dayjs.extend(advanced);
 
-type EmailAttachmentData = {
-  data: string;
-  name: string;
-  type: string;
-};
-
 type listingInfo = {
   id: string;
   name: string;
   juris: string;
 };
 
+const sesClient = new aws.SESClient({
```

Comment:
Curious where in this file you think these should live? @YazeedLoonat

Reply:
thats a great question, I think moving this and the below `const transporter = ...` into the constructor() makes the most sense

And then make `transporter` an instance variable like `polyglot`